account_asset_management/models/account_asset.py

```diff
@@ -895,7 +895,7 @@ def _compute_depreciation_table(self):
         asset_date_start = datetime.strptime(self.date_start, '%Y-%m-%d')
         fy = company.find_daterange_fy(asset_date_start)
         fiscalyear_lock_date = company.fiscalyear_lock_date
-        if fiscalyear_lock_date > self.date_start:
+        if fiscalyear_lock_date >= self.date_start:
```

Comment:
@luc-demeyer Thanks for the fix.

BTW, why don't you use [compute_fiscalyear_dates](https://github.com/OCA/account-financial-tools/blob/29d69c94b71c2d7f3e277dfca68b7f2ed71ce2e4/account_fiscal_year/models/res_company.py#L27) to get the fiscal year end date? That methods supports the standard Odoo method (12 month FY) and is improved when account_fiscal_year is installed.

Reply:
@sbidoul 
The 'compute_fiscalyear_dates' falls back on the standard odoo method when no date_range of type fiscalyear is defined. That logic is not correct since it relies on the single fiscalyear_last_month/day fields whereas a company may have multiple changes in fiscal year duration.
We should as a consequence need to fix the logic in the account_fiscalyear OCA module. Maybe a good idea for the OCA code sprint next week (akim will work with me on the 11.0 port of the account_asset_management modules next week hence we could add this into the 11.0 port, @grindtildeath is this ok for you ?).